CONTRIBUTION.md

```diff
@@ -0,0 +1,101 @@
+# Contribute to DB4S
+
+Thanks for your interest in our project.  
+This document is intended to serve as lighthouse for contributors, not an executor.
+
+## Contents
+- [How to ask about DB4S](#how-to-ask-about-db4s)
+- [How to contribute?](#how-to-contribute)
+    - [Report bug](#report-bug)
+    - [Suggest enhancement](#suggest-enhancement)
+    - [Translation](#translation)
+- [Styleguide](#styleguide)
+    - [CPP source code](#cpp-source-code)
+    - [Git commit messeage](#git-commit-message)
+
+## How to ask about DB4S?
+You have questions about our project! If so, please check the following first.
+
+- Look up related topics on the [wiki](https://github.com/sqlitebrowser/sqlitebrowser/wiki)
+- Search for existing similar issues
+
+If your question is still not resolved, you can create an issue and ask a question.  
+And also a template from GitHub issue will help you write good bug report.
+
+## How to contribute?
+### Report bug
+Please check the following before reporting a bug.
+
+- Look up ['Known Issues'](https://github.com/sqlitebrowser/sqlitebrowser/issues/1688)
+- Search for existing similar issues
+- Try in nightly build
```

Comment:
Add link to https://nightlies.sqlitebrowser.org/latest

Remove 'in' and use 'a'